app/grandchallenge/emails/templates/emails/email_detail.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %}
+{% load url %}
+{% load bleach %}
+
+{% block title %}{{ object.subject }} - Emails - {{ block.super }}{% endblock %}
+
+{% block breadcrumbs %}
+    <ol class="breadcrumb">
+        <li class="breadcrumb-item text-light">Emails</li>

```

Comment:
Maybe add a list view too so that people can find their emails back, both drafts and previously sent?